client/src/components/RelatedItemsAndOutfit/RelatedItemsAndOutfit.jsx

```diff
@@ -14,13 +14,16 @@ const RelatedItemsAndOutfit = ({ product, relatedProducts }) => {
           currentProduct={product}
           products={relatedProducts}
           carouselType={'related'}
+          onAction={() => console.log('show modal')}
```

Comment:
Does this console log need to be here?

Reply:
lol no, that was a placeholder. thank you!